tests/arc/timportedobj.nim

```diff
@@ -1,6 +1,7 @@
 discard """
   cmd: "nim c --gc:arc $file"
   action: "compile"
+  disabled: "windows"
```

Comment:
This test is not POSIX specific, but it can stay disabled on Windows for now. I'll make a follow-up PR that removes the `posix` module dependency.

Reply:
The follow-up PR: #1149.